src/main/java/com/internship/backend/mappper/LocationMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.internship.backend.mappper;
+
+import com.internship.backend.dto.LocationDTO;
+import com.internship.backend.model.Location;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Not sure if this class should be a component one, since it has only one method that parse from LocationDTO to Location model? Can the same behaviour be achieved using simple java class?